### PR TITLE
Hw 34.1/35.1 spring security

### PR DESCRIPTION
**Pull request by Valdes2**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
```
Comment (Sofasmile) [inline .travis.yml]: add empty line at the end of the file

File: src/main/java/mate/academy/spring/controller/RegistrationController.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+@Controller
+@RequestMapping("/user")
+public class RegistrationController {
+
+    @Autowired
+    private UserService userService;
+
+    @Autowired
+    DtoToEntity dtoToEntity;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/controller/RegistrationController.java]: add access modifier

File: src/main/java/mate/academy/spring/dto/DtoToEntity.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.dto;
+
+import mate.academy.spring.entity.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DtoToEntity {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/dto/DtoToEntity.java]:
```suggestion
public class DtoUtil {
```

File: src/main/java/mate/academy/spring/entity/Role.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "role_id")
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @ManyToMany(mappedBy = "roles")
+    private Set<User> users;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/entity/Role.java]: should role know something about user?

**Review by Sofasmile — COMMENTED** [review]
make small fixes

File: .travis.yml (hunk 1 above)
Comment (Valdes2) [reply .travis.yml]: fixed

File: src/main/java/mate/academy/spring/controller/RegistrationController.java (hunk 2 above)
Comment (Valdes2) [reply src/main/java/mate/academy/spring/controller/RegistrationController.java]: fixed

File: src/main/java/mate/academy/spring/dto/DtoToEntity.java (hunk 3 above)
Comment (Valdes2) [reply src/main/java/mate/academy/spring/dto/DtoToEntity.java]: fixed

File: src/main/java/mate/academy/spring/entity/Role.java (hunk 4 above)
Comment (Valdes2) [reply src/main/java/mate/academy/spring/entity/Role.java]: fixed

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**